blazor-features/OnChangeEventWithSelectDropdown/OnChangeEventWithSelectDropdown/Pages/Index.razor

```diff
@@ -0,0 +1,48 @@
+﻿@page "/"
+
+<PageTitle>Burger Configurator</PageTitle>
+
+<h1>Burger Configurator</h1>
+
+<p>Burger without topping: @BaseBurgerCost</p>
+<p>Topping: @SelectedTopping</p>
+<p>Total Cost: @TotalCost</p>
+
+<p>Please choose your topping from the list below:</p>
+
+<select class="select-element" @onchange="@handleChange">
+    @foreach (var topping in _toppings)
+    {
+        <option value="@topping.Id">@topping.Name  +$@topping.Price</option>
+    }
+</select>
+
+@code
+{
+    public record Topping(int Id, string Name, double Price);
+    public string? SelectedTopping;
+    public static double BaseBurgerCost = 5.4;
+    public double TotalCost = BaseBurgerCost;
+
+    public List<Topping> _toppings = new List<Topping>()
```

Comment:
@semirteskeredzic by convention, 
1. `private` variables can be written with the underscore prefix, but not `public` variables i.e `toppings`
2. variable names should be in `camelCase` and not `PascalCase`, i.e `selectedTopping`, `baseBurgerCost` `totalCost`,